code/modules/jobs/job_types/security_officer.dm

```diff
@@ -5,8 +5,8 @@
 	auto_deadmin_role_flags = DEADMIN_POSITION_SECURITY
 	department_head = list(JOB_HEAD_OF_SECURITY)
 	faction = FACTION_STATION
-	total_positions = 8 //Handled in /datum/controller/occupations/proc/setup_officer_positions() // SKYRAT EDIT: SET TO 8, WAS 5
-	spawn_positions = 8 //Handled in /datum/controller/occupations/proc/setup_officer_positions() // SKYRAT EDIT: SEE ABOVE
+	total_positions = 5 //Handled in /datum/controller/occupations/proc/setup_officer_positions() // SKYRAT EDIT: SET TO 5, WAS 5
+	spawn_positions = 5 //Handled in /datum/controller/occupations/proc/setup_officer_positions() // SKYRAT EDIT: SEE ABOVE
```

Comment:
This is essentially a revert if you're going back to 5, which means you shouldn't need these comments anymore (but I would hang onto them until a maintainer has a say in the matter).

Reply:
> This is essentially a revert if you're going back to 5, which means you shouldn't need these comments anymore (but I would hang onto them until a maintainer has a say in the matter).

That's what I was thinking, didn't want to change too much.